pkg/webhook/membercluster/membercluster_validating_webhook_test.go

```diff
@@ -0,0 +1,141 @@
+package membercluster
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+
+	clusterv1beta1 "github.com/kubefleet-dev/kubefleet/apis/cluster/v1beta1"
+	"github.com/kubefleet-dev/kubefleet/pkg/utils"
+
+	fleetnetworkingv1alpha1 "go.goms.io/fleet-networking/api/v1alpha1"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+func TestHandleDeleteServiceExportValidation(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]struct {
+		networkingEnabled     bool
+		expectAllowed         bool
+		expectedMessageSubstr string
+		validationMode        clusterv1beta1.DeleteValidationMode
```

Comment:
```suggestion
		networkingEnabled     bool
		validationMode        clusterv1beta1.DeleteValidationMode
		wantAllowed         bool
		wantMessageSubstr string
```
prefer "want" instead of expected